MAINTAINERS.md

```diff
@@ -17,3 +17,29 @@
   and calling `sha256sum` on it.
 * Update the release notes with a workspace setup section for the new version,
   such as [this example](https://github.com/tweag/rules_sh/releases/tag/v0.2.0)
+* Add the new version to the Bazel Central Registry as described below
+
+## Add a new version to the Bazel Central Registry
+
+* Follow the instructions given in the [README][bcr-add] to add a new version
+  of this module. Use the following input file to `add_module.py` for
+  reference - don't forget to update the version and dependencies:
+  ```
+  {
+      "build_file": null,
```

Comment:
I do not know if that is intended, but...
Perhaps we could retain build targets in the example inputfile, so that newcomers may easily reproduce the setup from ground up?
`"build_targets": []" => "build_targets": ["@rules_sh//..."]`

Reply:
This is used by [`add_module.py`](https://github.com/bazelbuild/bazel-central-registry/blob/2f3529db97074c69c03e992f6d6943fbc98d4fa4/tools/add_module.py#L112) in case no `presubmit.yml` is specified. However, in this case we specify one explicitly below, so the `build_targets` field ends up unused.